lab_prog_4.cpp

```diff
@@ -1,6 +1,6 @@
 #include<iostream>
 using namespace std;
-void add(int ,int);
+void add(int ,int);//function overloading
 void add(double ,double);
```

Comment:
This function returns void, But in definition of this function it returns zero. Explain why?

Reply:
here , the return type of  userdefined function is void so it is not returning anything but the main function has int as its return type so main function have to return a value.